gui/wxpython/lmgr/toolbars.py

```diff
@@ -68,71 +62,6 @@ def _toolbarData(self):
                                      ))
 
 
-class LMDataToolbar(BaseToolbar):
```

Comment:
Unnecessary large change in the code layout. DisplayPanelToolbar is more or less LMDataToolbar with addition of stuff from LMVectorToolbar, so why new class on a new place. Just put DisplayPanelToolbar in the same place as LMDataToolbar.